api/nodes.go

```diff
@@ -601,6 +604,67 @@ type HostDiskStats struct {
 	InodesUsedPercent float64
 }
 
+// DeviceGroupStats contains statistics for each device of a particular
+// device group, identified by the vendor, type and name of the device.
+type DeviceGroupStats struct {
```

Comment:
I copied the types here verbatim from plugins package.  Not sure if there are conventions we follow when doing that.

Reply:
Nope this is good